index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/styles.css">
+
+    <title>Awesome Books</title>
+</head>
+
+<body>
+    <main class="content">
+        <h1>Awesome Books</h1>
+        <div class="table" id ="booksTable" >
+
+        </div>
+        <form>
+            <div class="field">
+              <input type="text"
+                class="form-control" name="booktitle" id="booktitle" aria-describedby="Title" placeholder="Title">
+            </div>
+            <div class="field">
+              <input type="text"
+                class="form-control" name="bookauthor" id="bookauthor" aria-describedby="Author" placeholder="Author">
+            </div>
+                <input type="submit" class="form-control button" name="addbutton" id="addbutton" aria-describedby="Add" value="Add">
+        </form>
```

Comment:
- [ ] Kindly make sure to validate your form so the user can be able to add a new book with no `author` or `title`

    - ### Local storage is storing a book with no `title` or `author`
   ![image](https://user-images.githubusercontent.com/87197412/199325412-49f0f97a-45b8-4559-972c-22be99fe9046.png)